setup.cfg

```diff
@@ -30,6 +30,15 @@ spacy_scorers =
     spacy-experimental.tokenizer_senter_scorer.v1 = spacy_experimental.char_tokenizer.scorers:make_tokenizer_senter_scorer
 spacy_tokenizers =
     spacy-experimental.char_pretokenizer.v1 = spacy_experimental.char_tokenizer.char_pretokenizer:char_pretokenizer_v1
+spacy_misc =
+    spacy-experimental.subtree_suggester.v1 = spacy_experimental.span_suggesters.subtree_suggester:build_subtree_suggester
+    spacy-experimental.ngram_subtree_suggester.v1 = spacy_experimental.span_suggesters.subtree_suggester:build_ngram_subtree_suggester
+    spacy-experimental.chunk_suggester.v1 = spacy_experimental.span_suggesters.chunk_suggester:build_chunk_suggester
+    spacy-experimental.ngram_chunk_suggester.v1 = spacy_experimental.span_suggesters.chunk_suggester:build_ngram_chunk_suggester
+    spacy-experimental.sentence_suggester.v1 = spacy_experimental.span_suggesters.sentence_suggester:build_sentence_suggester
+    spacy-experimental.ngram_sentence_suggester.v1 = spacy_experimental.span_suggesters.sentence_suggester:build_ngram_sentence_suggester
```

Comment:
Since this is experimental, my feeling is more that it's okay to provide a wider/noisy range of things to experiment with easily, but we wouldn't necessarily do the same in spacy later.

Reply:
I think it's good to have at least some of these as an example of how it's done? I also agree that we should change it when migrating them to spaCy later.

Reply:
Ok, fair enough!